src/legacy/core_plugins/kibana/public/dashboard/top_nav/options.tsx

```diff
@@ -75,17 +82,9 @@ class OptionsMenuUi extends Component {
             data-test-subj="dashboardPanelTitlesCheckbox"
           />
         </EuiFormRow>
-
       </EuiForm>
     );
   }
 }
 
-OptionsMenuUi.propTypes = {
```

Comment:
Question: are we removing `propTyps` for components that we migrate to TypeScript?

Reply:
I have been, though I don't think we've come up with any official stance on it (I do recall it was brought up once before, I think in Canvas room... I think they decided to remove when in typescript... but I could also be remembering wrong cc @clintandrewhall ).

I like to get rid of it because it means two sources of truth and they can get out of sync.  Much easier to just have one to maintain, though I do realize that you then lose the runtime errors.